apps/admin_audit/lib/Actions/Files.php

```diff
@@ -36,134 +50,210 @@ class Files extends Action {
 	/**
 	 * Logs file read actions
 	 *
-	 * @param array $params
+	 * @param BeforeNodeReadEvent $event
 	 */
-	public function read(array $params): void {
+	public function read(BeforeNodeReadEvent $event): void {
+		try {
+			$params = [
+				'id' => $event->getNode()->getId(),
+				'path' => mb_substr($event->getNode()->getInternalPath(), 5),
+			];
+		} catch (InvalidPathException|NotFoundException $e) {
+			\OCP\Server::get(LoggerInterface::class)->error(
+				"Exception thrown in file read: ".$e->getMessage(), ['app' => 'admin_audit']
```

Comment:
```suggestion
				"Exception thrown in file read: ".$e->getMessage(), ['app' => 'admin_audit', 'exception' => $e]
```
Always provide the exception to the logger when you have it, it helps debugging.